eng/pipelines/templates/jobs/publish-coverage.yml

```diff
@@ -0,0 +1,74 @@
+parameters:
+  - name: ServiceDirectory
+    type: string
+    default: ''
+  - name: Artifacts
+    type: object
+    default: []
+  - name: DependsOn
+    type: string
+    default: ''
+  - name: RunCoverage
+    type: boolean
+    default: true
+
+
+jobs:
+  - job: Publish_Code_Coverage
+    pool:
+      name: azsdk-pool-mms-ubuntu-1804-general
+      vmImage: MMSUbuntu18.04
+
+    steps:
+      - task: UsePythonVersion@0
+        inputs:
+          versionSpec: '3.7'
+
+      # Download all existing artifacts (most of the time, it's whatever was uploaded before)
+      - pwsh: |
+          New-Item -Path $(Build.SourcesDirectory) -Name "_all_coverage_files" -ItemType "directory"
+        displayName: 'Create all coverages directory'
+        continueOnError: false
+        condition: and(succeededOrFailed(), ${{ parameters.RunCoverage }})
+
+      - task: DownloadPipelineArtifact@2
+        inputs:
+          source: current
+          path: '$(Build.SourcesDirectory)/_all_coverage_files'
+
+      - script: |
+          python -m pip install pip==20.1
```

Comment:
Can we instead place `pip==20.2` in the `eng/ci_tools.txt`? That way it's pinned in an expected location.